Sectors/Static/LON/Brecon.txt

```diff
@@ -1,42 +1,84 @@
                             N051.47.51.000 W001.41.36.000 N051.41.26.000 W002.00.00.000
+                            N051.41.26.000 W002.00.00.000 N051.41.26.000 W002.00.00.000
                             N051.41.26.000 W002.00.00.000 N051.15.26.000 W002.00.00.000
+                            N051.15.26.000 W002.00.00.000 N051.15.26.000 W002.00.00.000
                             N051.15.26.000 W002.00.00.000 N051.21.09.000 W003.37.47.000
+                            N051.21.09.000 W003.37.47.000 N051.21.09.000 W003.37.47.000
                             N051.21.09.000 W003.37.47.000 N051.21.38.000 W003.42.32.000
+                            N051.21.38.000 W003.42.32.000 N051.21.38.000 W003.42.32.000
                             N051.21.38.000 W003.42.32.000 N051.36.53.000 W006.12.08.000
+                            N051.36.53.000 W006.12.08.000 N051.36.53.000 W006.12.08.000
                             N051.36.53.000 W006.12.08.000 N051.40.34.000 W006.00.27.000
+                            N051.40.34.000 W006.00.27.000 N051.40.34.000 W006.00.27.000
                             N051.40.34.000 W006.00.27.000 N052.00.49.000 W006.07.20.000
+                            N052.00.49.000 W006.07.20.000 N052.00.49.000 W006.07.20.000
                             N052.00.49.000 W006.07.20.000 N052.09.31.000 W005.50.32.000
+                            N052.09.31.000 W005.50.32.000 N052.09.31.000 W005.50.32.000
                             N052.09.31.000 W005.50.32.000 N052.20.00.000 W005.30.00.000
+                            N052.20.00.000 W005.30.00.000 N052.20.00.000 W005.30.00.000
                             N052.20.00.000 W005.30.00.000 N052.57.05.000 W005.30.00.000
+                            N052.57.05.000 W005.30.00.000 N052.57.05.000 W005.30.00.000
                             N052.57.05.000 W005.30.00.000 N053.10.35.000 W005.30.00.000
+                            N053.10.35.000 W005.30.00.000 N053.10.35.000 W005.30.00.000
                             N053.10.35.000 W005.30.00.000 N053.11.48.000 W005.30.00.000
 
                             N052.17.02.000 W002.12.51.000 N052.05.26.000 W002.30.50.000
+                            N052.05.26.000 W002.30.50.000 N052.05.26.000 W002.30.50.000
                             N052.05.26.000 W002.30.50.000 N052.02.27.000 W002.36.35.000
+                            N052.02.27.000 W002.36.35.000 N052.02.27.000 W002.36.35.000
                             N052.02.27.000 W002.36.35.000 N052.00.29.000 W002.47.17.000
+                            N052.00.29.000 W002.47.17.000 N052.00.29.000 W002.47.17.000
                             N052.00.29.000 W002.47.17.000 N051.59.42.000 W003.36.01.000
+                            N051.59.42.000 W003.36.01.000 N051.59.42.000 W003.36.01.000
                             N051.59.42.000 W003.36.01.000 N051.56.26.000 W003.36.13.000
+                            N051.56.26.000 W003.36.13.000 N051.56.26.000 W003.36.13.000
```

Comment:
Just checking - you seem to have added loads of lines in this file where start and end points are identical, is this deliberate?